roles/ohpc_matlab/templates/r2018a

```diff
@@ -0,0 +1,20 @@
+#%Module
+
+set ver r2018a
+set matlabroot /opt/ohpc/pub-master/apps/matlab/M2/
+set url http://docs.uabgrid.uab.edu/wiki/MatLab
+set msg "This module adds Matlab $ver to various paths\n\nSee $url for usage examples\n"
+
+proc ModulesHelp { } {
+   puts stderr $msg
+}
+
+module-whatis   $msg
+
+#module load foss/2016b
+
+setenv MATLAB $matlabroot
+setenv MATLABROOT $matlabroot
+setenv MATLAB_HOME $matlabroot
+setenv MLM_LICENSE_FILE "27000@lmgr.rc.uab.edu"
+prepend-path PATH $matlabroot/bin
```

Comment:
This has MLM_LICENSE_FILE env var value has no place in the dev repo.  It is specific to our prod environment.

Likewise with the earlier url pointing at our prod documentation.